src/main/java/org/jabref/gui/customjfx/InputMethodSupport.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
```

Comment:
No. It is GPLv2. Not compatible. However, the complete JDK has the [linking exception](https://en.wikipedia.org/wiki/GPL_linking_exception). See http://hg.openjdk.java.net/openjfx/8/master/file/b80799b32c69/LICENSE#l326.

To avoid discussions with Linux distributions, please make a separate repository `org.jabref.gui.customjfx` with the same license file, a README.md and a gradle build script. The resulting JAR is then copied into https://github.com/JabRef/jabref/tree/master/lib.